README.md

```diff
@@ -10,4 +13,14 @@ markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
 
-$T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+$g(n) \in O(f(n)) \iff \exists c, n_0: g(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+Using change of base:
+
+$log_5(n) = log_2(n)/{log_2(5)}$
+
+Using the big O definition, we must prove $g(n) \leq c \cdot f(n)$, we can set $c = 1/{log_2(5)}$  , $g(n) = log_5(n)$  and $f(n) = log_2(n)$ :
```

Comment:
You can't assume a particular $g(n)$. It doesn't matter what you call it, you can't assume any particular expression.

Reply:
You must show this for all possible $g(n)$.